packages/api-bindings/src/apollo/cache/createProfileTypePolicy.ts

```diff
@@ -73,9 +71,9 @@ export function createProfileTypePolicy(
 
       followStatus: {
         read(_, { readField }) {
-          const activeWallet = activeWalletVar();
+          const session = getSession();
 
-          if (!activeWallet) {
+          if (!session || session.isNotAuthenticated()) {
```

Comment:
what could be the use case where `getSession` return a null value? 
Should we make it an `invariant` as it should never happen if the app is properly wired together?

Reply:
If everything is wired correctly requests requiring the session should never fire. Maybe it's a big assumption that all hooks (present and future) will be bug free from day-one.

These invariant are difficult to catch unless we increase our test coverage (not all read hooks do have an integration unit test). Also these field policies are triggered by profile/publication requests done in presenters. I would need to think about it if it's possible for those to fire before we know current session state.